meta/Observables.m

```diff
@@ -27,7 +27,23 @@
 FSObservables = { aMuon, aMuonUncertainty, aMuonGM2Calc, aMuonGM2CalcUncertainty,
                   CpHiggsPhotonPhoton, CpHiggsGluonGluon,
                   CpPseudoScalarPhotonPhoton, CpPseudoScalarGluonGluon,
-                  EDM, BrLToLGamma, FToFConversionInNucleus, bsgamma };
+                  EDM, BrLToLGamma, bsgamma };
+
+Module[{file, str, out, yes = 0, no = 0},
```

Comment:
Ouch! Parsing the `Makefile` is quite an ugly hack! :)

Do you need the information about the chosen loop libraries? If so, I think a cleaner way would be to define
~~~
FSLoopLibraries = { FSSOFTSUSY };
~~~
at the top of `meta/FlexibleSUSY.m`.

In a second step, I'd propose that the `configure` script appends the following line to the `models/<model>/FlexibleSUSY.m` file, which contains the information about the configured loop libraries:
~~~
FSLoopLibraries = { FSSOFTSUSY, FSCOLLIER, FSFFLite, FSLoopTools };
~~~
In `meta/Observables` you can then simply check which symbols are within `FSLoopLibraries` and enable/disable observables.

Reply:
@Expander, the code does not need information about enabled loop libraries. 

This ugly parser is for `FormCalc` and `FeynArts`. I am sorry, if it was not clear from the description of 18b249b. 
`Observables.m` is loaded before any variable inside `FlexibleSUSY.m` is defined. So this parser is the cheapest way to generate the code, which gives desired result: avoid generation of `FA+FC` dependent code in case they are disabled.

Reply:
By the way, this looks like a possible solution for #166 .